socorro/external/rabbitmq/crashstorage.py

```diff
@@ -210,14 +211,20 @@ def _consume_acknowledgement_queue(self):
             pass  # nothing to do with an empty queue
 
     #--------------------------------------------------------------------------
-    def _transaction_ack_crash(self, connection, acknowledgement_token):
-        self.config.logger.debug(
-            'RabbitMQCrashStorage acking with delivery_tag %s',
-            acknowledgement_token.delivery_tag
-        )
+    def _transaction_ack_crash(
+        self,
+        connection,
+        crash_id,
+        acknowledgement_token
+    ):
         connection.channel.basic_ack(
             delivery_tag=acknowledgement_token.delivery_tag
         )
+        self.config.logger.debug(
```

Comment:
I specifically moved the logging to after the call so that the log would happen only if the call did not raise an exception.  That way I count successes, not attempts.